policy-ci.tf

```diff
@@ -0,0 +1,7 @@
+###############################################################################
+# Policy: CI
+# ----------
+# A CI user has all the privileges of a developer and presently has no
+# additional permissions, so this file remains empty for future use.
+###############################################################################
+
```

Comment:
I wonder if it  might be better to define the policy with empty permissions, just to encourage people to  add CI  specific permissions should they encounter  them

Reply:
That's what I originally wanted to do and had! Unfortunately, having an empty policy blows up during `apply` (although weirdly not during `plan`).

Reply:
This is what I originally tried (it's the removal commit): https://github.com/FormidableLabs/terraform-aws-serverless/pull/22/commits/c1ac9713eb3c60fa3155a99827dd7ae19bdcf27c

If you've got other ideas, I'm game!